static/app/stores/Commiters/CommittersContext.tsx

```diff
@@ -0,0 +1,279 @@
+import {createContext, useContext, useEffect, useReducer} from 'react';
```

Comment:
we usually use `camelCase` when naming folders and files. Maybe we should still be consistent and bring this topic to the frontend tsc?

Reply:
I'll update this in a followup PR, I made a mistake